continuity.session.logs/src/main/java/org/continuity/session/logs/managers/SessionLogsPipelineManager.java

```diff
@@ -58,6 +60,22 @@ public String runPipeline() {
 		}
 	}
 
+
+	/**
+	 * Runs the pipeline using the session logs modularization. Based on the environment variable,
+	 * different input data is used.
+	 * 
+	 *
+	 * @return
+	 */
+	public String runPipeline(Map<String, String> hostnames) {
+		if (USEOPENxtrace) {
```

Comment:
Just as an idea: We could add an option to the order defining the type of data the link points to. Then, we can use it to determine which implementation to execute instead of configuring it statically.
However, this is not really part of this story...

Reply:
Note: Will be implemented